CONTRIBUTING.md

```diff
@@ -93,6 +93,13 @@ test-app$ yarn install
 test-app$ yarn watch
 ```
 
+### Test Options
+
+| Env                 | Description                                                                                                           |
+| ------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| TEST_DEPLOY_TARGET  | Deploy to `AWS` (with default credential chain) or `LOCALSTACK`, Default: `LOCALSTACK`                                |
+| TEST_DESTROY_STACKS | When truthy, will destroy any stacks created when the runtime test suite completes, Default: False (unless `CI=true`) |
```

Comment:
Curious if we can trigger this when the PR is closed? That way new commits trigger updates to the stack? Or is that too risky? What's the plan for isolation?

Reply:
I was thinking of using the self-destructing stack (via step functions) where a machine starts and decided if it should destruct the stack based on the last deployment time. That way, an active PR will maintain it's stack and a stale one will not use up resources (with a higher cold start time).

Going to play with this today.

Reply:
What would the timeout be on that step function?

Reply:
The default I have right now is 12 hours, but we can adjust to anything under a year.